spec/services/usps_in_person_proofing/enrollment_helper_spec.rb

```diff
@@ -60,19 +60,37 @@
           expect(applicant.state).to eq(Idp::Constants::MOCK_IDV_APPLICANT[:state])
           expect(applicant.zip_code).to eq(Idp::Constants::MOCK_IDV_APPLICANT[:zipcode])
           expect(applicant.email).to eq('no-reply@login.gov')
-          expect(applicant.unique_id).to be_a(String)
+          expect(applicant.unique_id).to eq(enrollment.unique_id)
 
           proofer.request_enroll(applicant)
         end
 
         subject.schedule_in_person_enrollment(user, pii)
       end
 
-      it 'sets enrollment status to pending and sets enrollment established at date' do
+      context 'when the enrollment does not have a unique ID' do
+        it 'uses the deprecated InPersonEnrollment#usps_unique_id value to create the enrollment' do
```

Comment:
Ohhhhh, okay, this spec answers my previous question in a comment on `enrollment_helper.rb`. 👍🏻